units/systemd-networkd.socket

```diff
@@ -15,7 +15,7 @@ DefaultDependencies=no
 Before=sockets.target
 
 [Socket]
-ReceiveBuffer=8M
+ReceiveBuffer=128M
```

Comment:
Is this memory permanently reserved, or on demand?

Reply:
It's a limit, not more. i.e. when nothing is in the buffer no memory is allocated for it. And if there's little in the buffer only little is allocated.